src/journey-maps/angular/components/feature-event-listener/feature-event-listener.ts

```diff
@@ -255,11 +274,66 @@ export class SbbFeatureEventListener implements OnChanges, OnDestroy {
     }
   }
 
+  // this method seems to return true if any of the clicked features is selected and has the same
+  // SbbFeatureDataType as the top-most clicked feature.
+  private _anySelectedClickedFeatureHasSameTypeAsTopClickedFeature(
+    clickedFeatures: SbbFeatureData[],
+    topClickedFeatureType: SbbFeatureDataType,
+    selectedFeatures?: { features: SbbFeatureData[] | undefined } | undefined
```

Comment:
```suggestion
    selectedFeatures?: { features: SbbFeatureData[] | undefined }
```

Nitpick, aber ist obsolet wegen des `?` hinter `selectedFeatures`